docs/serving/autoscaling/scale-bounds.md

```diff
@@ -7,6 +7,8 @@ type: "docs"
 
 To apply upper and lower bounds to autoscaling behavior, you can specify scale bounds in both directions.
 
+You can also specify the initial scale a revision is scaled to immediately after creation, either as a default for all revisions, or for a specific revision using an annotation.
```

Comment:
```suggestion
You can also specify the initial scale that a Revision is scaled to immediately after creation.
This can be a default configuration for all Revisions, or for a specific Revision using an annotation.
```